### PR TITLE
Multiple fixes and code formatting

### PR DESCRIPTION
**Pull request by NefixEstrada**
Hello again!

I've fixed some issues with the existing functions (missing edge cases mainly), and written some texts!

Also, I noticed that there was a linter but wasn't enforcing anything, so I switched to black and formatted the whole repo.

If you don't like the formatting part, I can remove it, but I thought it would be nice to enforce the same style always!

Thanks

### CODE REVIEW

**Comment by Inveracity** [comment]
Nice! I will give this a look as soon as I can!

And yes I totally prefer Black as well 👍

**Comment by Inveracity** [comment]
@NefixEstrada I'm not sure I resolved the conflicts correctly, do you mind taking a look?

**Review by NefixEstrada — COMMENTED** [review]
Sorry for the late answer, I was on vacation

File: rethinkdb_mock/db.py [hunk 1]
```diff
@@ -3,99 +3,80 @@
...
 from future.utils import iteritems
 import rethinkdb
+import rethinkdb.ast as r_ast
 
 from . import rtime
 from . import util
-from .ast_base import BinExp
 from .rql_rewrite import rewrite_query
+from .ast_base import BinExp
 from .rql_rewrite import RQL_TYPE_TRANSLATIONS
```
Comment (NefixEstrada) [inline rethinkdb_mock/db.py:12]: I don't think those two imports are required now. In fact, I think that I deleted `RQL_TYPE_TRANSLATIONS` as a whole

**Comment by Inveracity** [comment]
> from .rql_rewrite import RQL_TYPE_TRANSLATIONS

hope you had a nice vacation!

> I don't think those two imports are required now. In fact, I think that I deleted `RQL_TYPE_TRANSLATIONS` as a whole

ah yeah I see it now! thank you

**Pull request merged**